extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -113,8 +113,8 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 	try {
 		const parentUrn = context.nodeInfo ? await getParentUrn(context) : undefined;
 		const objectType = context.nodeInfo ? context.nodeInfo.nodeType as ObjectManagement.NodeType : (context.connectionProfile.databaseName === '' ? ObjectManagement.NodeType.Server : ObjectManagement.NodeType.Database);
-		const objectName = context.nodeInfo ? context.nodeInfo.label : objectManagementLoc.PropertiesHeader;
-		const objectUrn = context.nodeInfo ? context.nodeInfo!.metadata!.urn : undefined;
+		const objectName = context.nodeInfo ? context.nodeInfo.label : (context.connectionProfile.databaseName === '' ? context.connectionProfile.serverName : context.connectionProfile.databaseName);
```

Comment:
I'd suggest doing 

`!context.connectionProfile.databaseName ? context.connectionProfile.serverName : context.connectionProfile.databaseName`

Instead since databaseName can be undefined.